mlir/lib/Dialect/QC/Translation/qasm3/QASM3Emitter.cpp

```
@@ -0,0 +1,1477 @@
+
+  /// Bind a floating-point variable @p id to the value of @p value.
+  LogicalResult assignFloat(StringRef id, const Expr& value) {
+    auto emitted = emitFloat(value);
+    if (failed(emitted)) {
+      return failure();
+    }
+    floatValues.insert(id, *emitted);
+    return success();
+  }
+
+  LogicalResult qubitRegister(SMLoc /*loc*/, StringRef id, const Expr* size) {
```

Cppcheck (2.21.0)

[style] The function 'qubitRegister' is never used.

(unusedFunction)